workflows/nallo.nf

```diff
@@ -518,6 +518,19 @@ workflow NALLO {
                     }
                 }
             }
+            if (params.skip_phasing_wf) {
```

Comment:
Alternative: Remove call to `METHYLATION above` and select input something like this:
```groovy
ch_methylation_in = params.skip_phasing_wf ? MINIMAP2 : PHASING
```

Reply:
Also note that if you run with parallel_alignments > 1, or there are more than one input file per sample, the output from minimap needs to be merged. Use the (not so well described) channels on line 226-228 for input instead: https://github.com/genomic-medicine-sweden/nallo/blob/9000326c61f0b100c77419fec2b87beabfa9e810/workflows/nallo.nf#L226-L228